pkg/config/config_test.go

```diff
@@ -206,8 +254,16 @@ func TestIsEnabled(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.Name, func(t *testing.T) {
-			if IsEnabled(test.Org, test.Repo, "thisrepo") != test.Expect {
-				t.Errorf("Unexpected results. Expected: %v", test.Expect)
+			get = func(context.Context, string, string) (*github.Repository,
+				*github.Response, error) {
+				b := test.Org.OptOutPrivateRepos
+				return &github.Repository{
+					Private: &b,
```

Comment:
This should be a separate input for each test, the mock response from github. Then the org config as you have it, then the expected output.

Reply:
Thanks. Added mock config with d86295d7a81c9f9bdf84963965e9777719afa404.